### ISSUE TITLE
Display problem with Chinese String

### ISSUE THREAD

**Issue by suihanhbr** [gentics/mesh#610]
## Gentics Mesh Version, operating system, or hardware.
- v0.28.2

## Operating System
- Windows

## JVM
- Oracle JDK 1.8.0_191

## Problem

I add  a category or filefolder with Chinese string to demo project ,for exmaple :测试, it can't be displayed in the node list of the demo project, But it can be seen at the website which is the instance of mesh-vertx-example.
According of my test, the problem occurs on Windows, but do not happened on Mac.

## Reproducer
Test on the demo with any Chinese string.
## Expected behaviour and actual behaviour
Expected behaviour: the Chinese item can be displayed on the node list of the demo project and can be accessed by website.
actual behaviour: the Chinese item don't be seen on the node list of the demo project, but can be accessed by website.

**Comment by Jotschi** [comment]
We think that this is just an issue with the demo app or at least the font. Thanks for reporting. We'll try to find a workaround / solution.

**Comment by suihanhbr** [comment]
I test it with v0.29.4 ,and it still don't work. here is the bug report, I hope it can help.
```
[2019-01-30T13:01:40,387][DEBUG][o.e.a.b.TransportShardBulkAction] [mesh-node-217f8c981ada4642bf8c981adaa642c3-54873d2a243a4d27873d2a243a8d2764-87d5b68c7ec845d095b68c7ec895d030-published][4] failed to execute bulk item (index) BulkShardRequest [[mesh-node-217f8c981ada4642bf8c981adaa642c3-54873d2a243a4d27873d2a243a8d2764-87d5b68c7ec845d095b68c7ec895d030-published][4]] containing [index {[mesh-node-217f8c981ada4642bf8c981adaa642c3-54873d2a243a4d27873d2a243a8d2764-87d5b68c7ec845d095b68c7ec895d030-published][default][0147c7623a6e4fff87c7623a6efffffa-en], source[{"uuid":"0147c7623a6e4fff87c7623a6efffffa","editor":{"uuid":"e5861ba26b914b21861ba26b91ab211a"},"edited":"2019-01-30T05:01:38Z","creator":{"uuid":"e5861ba26b914b21861ba26b91ab211a"},"created":"2019-01-30T05:01:37Z","project":{"name":"demo","uuid":"217f8c981ada4642bf8c981adaa642c3"},"tags":{"name":[],"uuid":[]},"tagFamilies":{},"_roleUuids":["cc9999a4194d4f069999a4194d1f06d7","ddcddceb33d648318ddceb33d618314f","be703d9cccdf4771b03d9cccdff771c1","cc9999a4194d4f069999a4194d1f06d7","be703d9cccdf4771b03d9cccdff771c1"],"parentNode":{"uuid":"110845c6a2a34eeb8845c6a2a38eeb11"},"language":"en","schema":{"name":"category","uuid":"2ca2362b041247c4a2362b041227c4da","version":"1.0"},"fields":{"name":"����һ��","slug":"����һ��"},"displayField":{"key":"name","value":"����һ��"},"version":"89f6ad6b"}]}]
org.elasticsearch.index.mapper.MapperParsingException: failed to parse [fields.name]
	at org.elasticsearch.index.mapper.FieldMapper.parse(FieldMapper.java:302) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.parseObjectOrField(DocumentParser.java:485) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.parseValue(DocumentParser.java:607) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.innerParseObject(DocumentParser.java:407) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.parseObjectOrNested(DocumentParser.java:384) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.parseObjectOrField(DocumentParser.java:482) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.parseObject(DocumentParser.java:500) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.innerParseObject(DocumentParser.java:394) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.parseObjectOrNested(DocumentParser.java:384) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.internalParseDocument(DocumentParser.java:93) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:67) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:261) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.shard.IndexShard.prepareIndex(IndexShard.java:708) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.shard.IndexShard.applyIndexOperation(IndexShard.java:686) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.shard.IndexShard.applyIndexOperationOnPrimary(IndexShard.java:667) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.bulk.TransportShardBulkAction.executeIndexRequestOnPrimary(TransportShardBulkAction.java:548) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.bulk.TransportShardBulkAction.executeIndexRequest(TransportShardBulkAction.java:140) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.bulk.TransportShardBulkAction.executeBulkItemRequest(TransportShardBulkAction.java:236) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.bulk.TransportShardBulkAction.performOnPrimary(TransportShardBulkAction.java:123) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.bulk.TransportShardBulkAction.shardOperationOnPrimary(TransportShardBulkAction.java:110) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.bulk.TransportShardBulkAction.shardOperationOnPrimary(TransportShardBulkAction.java:72) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryShardReference.perform(TransportReplicationAction.java:1033) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryShardReference.perform(TransportReplicationAction.java:1011) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.ReplicationOperation.execute(ReplicationOperation.java:104) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.onResponse(TransportReplicationAction.java:358) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.onResponse(TransportReplicationAction.java:298) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$1.onResponse(TransportReplicationAction.java:974) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$1.onResponse(TransportReplicationAction.java:971) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.shard.IndexShardOperationPermits.acquire(IndexShardOperationPermits.java:238) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.shard.IndexShard.acquirePrimaryOperationPermit(IndexShard.java:2211) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction.acquirePrimaryShardReference(TransportReplicationAction.java:983) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction.access$500(TransportReplicationAction.java:97) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.doRun(TransportReplicationAction.java:319) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:294) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:281) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:652) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:637) [elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.1.2.jar:6.1.2]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:1.8.0_191]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:1.8.0_191]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_191]
Caused by: com.fasterxml.jackson.core.JsonParseException: Invalid UTF-8 start byte 0xb2
 at [Source: org.elasticsearch.common.bytes.BytesReference$MarkSupportingStreamInputWrapper@768d8dfd; line: 1, column: 699]
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1702) ~[jackson-core-2.8.10.jar:2.8.10]
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:558) ~[jackson-core-2.8.10.jar:2.8.10]
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._reportInvalidInitial(UTF8StreamJsonParser.java:3544) ~[jackson-core-2.8.10.jar:2.8.10]
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._reportInvalidChar(UTF8StreamJsonParser.java:3538) ~[jackson-core-2.8.10.jar:2.8.10]
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._finishString2(UTF8StreamJsonParser.java:2543) ~[jackson-core-2.8.10.jar:2.8.10]
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._finishAndReturnString(UTF8StreamJsonParser.java:2469) ~[jackson-core-2.8.10.jar:2.8.10]
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser.getText(UTF8StreamJsonParser.java:315) ~[jackson-core-2.8.10.jar:2.8.10]
	at org.elasticsearch.common.xcontent.json.JsonXContentParser.text(JsonXContentParser.java:83) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.common.xcontent.support.AbstractXContentParser.textOrNull(AbstractXContentParser.java:237) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.TextFieldMapper.parseCreateField(TextFieldMapper.java:345) ~[elasticsearch-6.1.2.jar:6.1.2]
	at org.elasticsearch.index.mapper.FieldMapper.parse(FieldMapper.java:297) ~[elasticsearch-6.1.2.jar:6.1.2]
	... 42 more
13:01:40.602 [Linear Magcargo] ERROR [OkHttp http://localhost:9200/...] - Could not store document {mesh-node-217f8c981ada4642bf8c981adaa642c3-54873d2a243a4d27873d2a243a8d2764-87d5b68c7ec845d095b68c7ec895d030-published:0147c7623a6e4fff87c7623a6efffffa-en} - mapper_parsing_exception : failed to parse [fields.name]
13:01:40.729 [Linear Magcargo] INFO  [OkHttp http://localhost:9200/...] - 0:0:0:0:0:0:0:1 - - [Wed, 30 Jan 2019 05:01:39 GMT] "POST /api/v1/demo/search/nodes?lang=en,de&page=1&perPage=10 HTTP/1.1" 200 6243 "http://localhost:8080/mesh-ui/" "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Safari/537.36"
13:01:41.051 [Linear Magcargo] INFO  [vert.x-eventloop-thread-2] - 0:0:0:0:0:0:0:1 - - [Wed, 30 Jan 2019 05:01:38 GMT] "POST /api/v1/demo/nodes/0147c7623a6e4fff87c7623a6efffffa/published?lang=en HTTP/1.1" 200 243 "http://localhost:8080/mesh-ui/" "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Safari/537.36"****
```

**Comment by Jotschi** [comment]
Duplicate of #498

**Comment by Jotschi** [comment]
@suihanhbr I was not able to reproduce the issue. Can you please post the ES version that you are using, your current locale and a request which reproduces the issue? I have tried to use GB2312 encoded strings and UTF-8 encoded text. Both work fine. Did you maybe use UTF16 encoded text?

**Comment by Jotschi** [comment]
Fix has been released with 0.36.2
